template/contract/Import.sol

```diff
@@ -1,11 +1,11 @@
 pragma solidity 0.6.12;
 
 // Listing all contracts used by the deploy scripts to force dapp.tools to build them
-import 'mip21-toolkit/tokens/RwaTokenFactory.sol';
-import 'mip21-toolkit/tokens/RwaToken.sol';
-import 'mip21-toolkit/urns/RwaUrn2.sol';
-import 'mip21-toolkit/conduits/RwaOutputConduit2.sol';
-import 'mip21-toolkit/jars/RwaJar.sol';
-import 'mip21-toolkit/oracles/RwaLiquidationOracle.sol';
-import 'dss-gem-joins/join-auth.sol';
-import 'forward-proxy/ForwardProxy.sol';
\ No newline at end of file
+import "mip21-toolkit/tokens/RwaTokenFactory.sol";
+import "mip21-toolkit/tokens/RwaToken.sol";
+import "mip21-toolkit/urns/RwaUrn2.sol";
+import "mip21-toolkit/conduits/RwaOutputConduit2.sol";
+import "mip21-toolkit/jars/RwaJar.sol";
+import "mip21-toolkit/oracles/RwaLiquidationOracle.sol";
+import "dss-gem-joins/join-auth.sol";
+import "forward-proxy/ForwardProxy.sol";
```

Comment:
This file is probably not necessary anymore.

Reply:
I will work on template in separate PR, there is still an old way of deployment